content/docs/user-guide/basic-concepts/workspace.md

```diff
@@ -6,3 +6,39 @@ tooltip: >-
   models, etc. Typically, it's also a Git repository. It will contain your DVC
   project.
 ---
+
+# Workspace
+
+A data science project consists of data obtained from many different sources.
+This data may be split into multiple files or directories or (as the project
+structure needs) have different versions for different requirements. e.g. A
+smaller / simplified version might be required in prototyping for faster
+feedback and shorter training times. A single workspace to manage all artifacts
+of a project is desirable, although versioning needs and managing dependencies
+make it increasingly complex.
+
+DVC allows a single directory to contain all your project artifacts. The
+workspace is the directory containing _user visible_ part of your
+<abbr>project</abbr> e.g. raw datasets, source code, ML models, etc. Users work
+in this directory using their data and model files that and manipulate the
+contents through DVC commands.
+
+Files and directories in the workspace can be added to DVC (`dvc add`) or they
+can be downloaded from external sources (`dvc get`, `dvc import`,
+`dvc import-url`). Changes to the files, directories, notebooks, models, and in
+general machine learning file system can be tracked (`dvc commit`) and versioned
```

Comment:
I know "machine learning file system" was a desired SEO term (i.e. something people search for) but the way it's used here doesn't make sense IMO. What are ML FSs? (Idk)

I would skip that part here ("and in general...") and look for another way to use that keyword, if possible.

Reply:
I moved the phrase to the previous paragraph and modified this in 44af0a9 and 42559e2

Reply:
Better! 👍